enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/util/TLSUtils.java

```diff
@@ -108,24 +110,36 @@ public static void addCertsToTruststore(KeyStore trustStore, String filePath) th
     }
 
     private static List<Certificate> getCertsFromFile(String filepath, boolean restrictToOne)
-            throws CertificateException, IOException {
-        try (FileInputStream fileInputStream = new FileInputStream(filepath);
-             BufferedInputStream bufferedInputStream = new BufferedInputStream(fileInputStream)) {
-            List<Certificate> certList = new ArrayList<>();
-            CertificateFactory cf = CertificateFactory.getInstance(X509);
-            int count = 0;
-            while (bufferedInputStream.available() > 0) {
-                if (count > 1 && restrictToOne) {
-                    log.warn("Provided PEM file contains more than one certificate. Hence proceeding with" +
-                            "the first certificate in the File");
+            throws CertificateException, IOException, EnforcerException {
+        String content = new String(Files.readAllBytes(Paths.get(filepath)));
+
+        if (!content.contains(endCertificateDelimiter)) {
+            throw new EnforcerException("Content Provided within the certificate file:" + filepath + "is invalid.");
+        }
+
+        int endIndex = content.lastIndexOf(endCertificateDelimiter) + 25;
```

Comment:
Shall we use a const for `25` may be getting the length of `endCertificateDelimiter`